src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
-
 // write your code here
+
+// Отримуємо таблицю
+
+const table = document.querySelector('table');
+
+// Отримуємо всі рядки в таблиці (включаючи шапку)
+
+const rows = table.querySelectorAll('tr');
+
+rows.forEach((row) => {
+  // Отримуємо другу колонку (індекс 1)
+  const secondColumm = row.children[1].cloneNode(true);
+
+  // Вставляємо клону стовпа перед останнім
```

Comment:
it is bad practice to write useless comments. 
Code shuld be understandable without comments.
General practice to write comments in English.